src/qibocal/tests/test_data.py

```diff
@@ -183,3 +193,68 @@ def test_get_values_data():
     """Test get_values method of Data class"""
     data = random_data(5)
     assert (data.get_values("int") == data.df["int"]).all()
+
+
+def test_save_open_data_units_csv():
+    """Test to_csv and load_data methods of DataUnits"""
+    folder = "test_folder/data/test_routine"
+    if not os.path.isdir("test_older"):
+        os.mkdir("test_folder")
+        os.mkdir("test_folder/data")
+        os.mkdir(folder)
```

Comment:
You can actually do this passage in a single line.
```suggestion
        os.makedirs(folder)
```